### ISSUE TITLE
Add Israel Provider

### ISSUE THREAD

**Issue by yotamN** [schildbach/public-transport-enabler#201]
Israel provide GTFS files for the whole country and it's already in Navita, it would be nice and hopefully simple to add it. I can try to create a pull request but I'm not sure how to check that it works

**Comment by mikwee** [comment]
Any updates?

**Comment by GPery** [comment]
A bit stale, but here is a repository that currently pulls GTFS and SIRI data from the Israeli Ministry of Transportation: https://github.com/hasadna/open-bus

**Comment by thePrivacyFanatic** [comment]
> A bit stale, but here is a repository that currently pulls GTFS and SIRI data from the Israeli Ministry of Transportation

yes, tough they do it in order to preserve a record so the implementation that they have may be a little different from what we need here.
 just to understand there is no branch for it as of yet?
